src/sonic-config-engine/minigraph.py

```diff
@@ -358,7 +365,10 @@ def parse_dpg(dpg, hname):
             for i, member in enumerate(vmbr_list):
                 vmbr_list[i] = port_alias_map.get(member, member)
                 sonic_vlan_member_name = "Vlan%s" % (vlanid)
-                vlan_members[(sonic_vlan_member_name, vmbr_list[i])] = {'tagging_mode': 'untagged'}
+                if len(intf_vlan_mbr[member]) > 1:
+                    vlan_members[(sonic_vlan_member_name, vmbr_list[i])] = {'tagging_mode': 'tagged'}
```

Comment:
If there are multiple vlans, does that mean, there are no untagged member? I mean, tagging determines the pvid of port. So all untagged packets on this port would be dropped. Is that the expectation?

Reply:
yes